recipes/pixman/all/test_package/CMakeLists.txt

```
@@ -1,8 +1,7 @@
 cmake_minimum_required(VERSION 3.1)
 project(test_package C)
 
-include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
-conan_basic_setup()
+find_package(pixman)
```

```suggestion
find_package(pixman CONFIG REQUIRED)
```
Enforce that not found is an error